### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**
Some typos found in https://github.com/mne-tools/mne-python/pull/11086

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -15441,6 +15511,8 @@ flexibilty->flexibility
 flexibily->flexibly, flexibility,
 flext->flex
 flie->file
+fliter->filter
+fliters->filters
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:15515]:
```text
flitered->filtered
flitering->filtering
```
* https://github.com/search?q=flitered&type=code  2720
* https://github.com/search?q=flitering&type=code  5568

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -17302,6 +17382,9 @@ icrease->increase
 icreased->increased
 icreases->increases
 icreasing->increasing
+icrement->increment
+icremented->incremented
+icrements->increments
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:17387]:
```text
icrementing->incrementing
icrementally->incrementally
```
* https://github.com/search?q=icrementing&type=code  490
* https://github.com/search?q=icrementally&type=code  146

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -20526,6 +20618,7 @@ mamento->memento
 mamentos->mementos
 mamory->memory
 mamuth->mammoth
+manafacturer->manufacturer
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:20621]:
```text
manafacturers->manufacturers
```
* https://github.com/search?q=manafacturers&type=code  790

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -22900,6 +22997,10 @@ obect->object
 obediance->obedience
 obediant->obedient
 obejct->object
+obejcted->objected
+obejction->objection
+obejctions->objections
+obejcts->objects
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:23003]:
```text
obejctive->objective
obejctives->objectives
obejctively->objectively
```
* https://github.com/search?q=obejctive&type=code  3759
* https://github.com/search?q=obejctives&type=code  304
* https://github.com/search?q=obejctively&type=code  5

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -25087,6 +25195,7 @@ positionns->positions
 positionof->position, position of,
 positiv->positive
 positivies->positives
+positivy->positivity, positivly, positive,
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:25198]: Suggested word `positivly` is a misspelling. Please help to correct it.
```text
positivy->positivity, positively, positive,
```
* https://github.com/search?q=positivy&type=code  297

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -27653,6 +27766,7 @@ reguster->register
 rehersal->rehearsal
 rehersing->rehearsing
 reicarnation->reincarnation
+reight->right
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:27769]: Please suggest `eight, freight,` as well
```text
reight->right, eight, freight,
```
* https://github.com/search?q=reight&type=code  12602

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -28360,6 +28477,7 @@ representated->represented
 representating->representing
 representd->represented
 represente->represents, represented,
+representes->represents
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:28480]: Please suggest `represented` as well. `s` key is beside `d`
```text
representes->represents, represented,
```
* https://github.com/search?q=representes&type=code  14633

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -33752,6 +33887,8 @@ tramsmitting->transmitting
 tranaction->transaction
 tranactional->transactional
 tranactions->transactions
+tranalation->translation
+tranalations->translations
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:33891]:
```text
tranalating->translating
```
* https://github.com/search?q=tranalating&type=code  7

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -34268,6 +34406,7 @@ truns->turns
 trustworthyness->trustworthiness
 trustworty->trustworthy
 trustwortyness->trustworthiness
+trustwothy->trustworthy
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:34409]:
```text
trustworthly->trustworthy
```
* https://github.com/search?q=trustworthly&type=code  241

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
Thanks @vikivivi. Done in [cd05278](https://github.com/codespell-project/codespell/pull/2481/commits/cd0527882d7fa50749e0a9e8d1ba088631ce793f)

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**